src/pybind/mgr/cephadm/inventory.py

```diff
@@ -508,6 +515,34 @@ def daemon_cache_filled(self):
         return all((h in self.last_daemon_update or h in self.mgr.offline_hosts)
                    for h in self.get_hosts())
 
+    def schedule_daemon_action(self, host: str, daemon_name: str, action: str):
+        priorities = {
+            'start': 1,
+            'restart': 2,
+            'reconfig': 3,
+            'redeploy': 4,
+            'stop': 5,
+        }
```

Comment:
minor, but I wonder if at some point we should refactor these to be on the `DaemonDescription` and track the scheduled action there ...